hearth/lib/hearth/http/api_error.rb

```diff
@@ -1,25 +0,0 @@
-# frozen_string_literal: true
```

Comment:
I think there's rbs for this file? Check to see if any other rbs needs updates.

Reply:
Good call removed and checked all other rbs.